recipes/erlang/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set version = "18.3" %}
+
+package:
+  name: erlang
+  version: {{ version }}
+
+source:
+  fn: otp_src_{{ version }}.tar.gz
+  url: http://erlang.org/download/otp_src_{{ version }}.tar.gz
+  sha256: "fdab8129a1cb935db09f1832e3a7d511a4aeb2b9bb3602ca6a7ccb9730d5c9c3"
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - readline
+    - openssl
+    - autoconf
```

Comment:
So, based on the docs we shouldn't need `autoconf` unless we are building from a non-release (e.g. git repo). Should be able to drop it.